recipe/meta.yaml

```diff
@@ -24,8 +24,8 @@ requirements:
   - bob.io.base
   - giflib
   - libpng
```

Comment:
If we go down that road we will end up pinning everything! We need a way to trigger re-builds when a new version is out.